api/institutions/views.py

```diff
@@ -1,11 +1,14 @@
 from rest_framework import generics
 from rest_framework import permissions as drf_permissions
+from rest_framework import status
+from rest_framework.response import Response
 
 from modularodm import Q
 
 from framework.auth.oauth_scopes import CoreScopes
 
-from website.models import Institution, Node, User
+from website.models import Node, User
+from website.project.model import Institution
```

Comment:
Not super important, but is there a reason the `Institution` model isn't included in `website.models`?